### PR TITLE
Update topology.py

### PR DESCRIPTION
**Pull request by Nangos**
added functionality of checking path equivalence

### CODE REVIEW

File: simulator/topology.py [hunk 1]
```diff
@@ -142,113 +142,122 @@ def draw_topology(self):
...
+        return shortest_path, shortest_length
 
-        correct_path = []
-        for i in range(len(shortest_path) - 1):
-            correct_path.append((shortest_path[i], shortest_path[i + 1]))
-        return correct_path
-
-    def get_tree_correct_path(self, source):
+    
+    # def get_tree_correct_path(self, source):
```
Comment (starzia) [inline simulator/topology.py:152]: please remove this.

**Review by starzia — APPROVED** [review]

**Comment by starzia** [comment]
@kaiyuhou  please review this PR.

**Comment by kaiyuhou** [comment]
I checked it with Guannan. It works fine.

**Comment by kaiyuhou** [comment]
Wait, we need a small modification on this version. Please ignore this pull request.

**Pull request closed without merging**